lib/lite_cable/server.rb

```diff
@@ -7,6 +7,8 @@ module LiteCable
   #
   # Inspired by https://github.com/ngauthier/tubesock/blob/master/lib/tubesock.rb
   module Server
+    # FIXME 'websocket' should be in gemspec by default - because whole
+    # litecable can't run without it
```

Comment:
I'm not sure why it not in gem deps by default, may I add It?
Because `require 'websocket'` fails if gem not exists, so a user has no choices except to add it to his Gemfile.

Reply:
Hm, I guess, `require 'websocket'` should be moved to `Middleware`. We don't need it for AnyCable, for example, so it's optional.

Reply:
Anyway, it crashes the app if 'websocket' is not in Gemfile, even if it in Middleware and we run Anycable - I think it because of `require 'iodine'` in lite_cable.rb and `require "lite_cable/server"` inside iodine.rb.

Reply:
Maybe it'll good solution to split iodine.rb in two pieces - one with `Litecable.iodine!` definition and one with implementation(integrations/iodine.rb, for example).
And call `require 'integrations/iodine.rb' directly inside `Litecable.iodine!` method, so it will not include unused gems. 

And also make more generic `Middleware` class, so it can be used in Iodine integration without requiring 'websocket'.

Reply:
Why not having a separate `Middleware` for Iodine? I guess, it should be totally different from the one we have.